db/patch68.sql

```diff
@@ -3,4 +3,4 @@
 -- This resolves https://joindin.jira.com/browse/JOINDIN-725
 ALTER DATABASE joindin CHARACTER SET utf8 COLLATE utf8_unicode_ci;
 
-INSERT INTO patch_history SET patch_number = 68;
\ No newline at end of file
+INSERT INTO patch_history SET patch_number = 69;
```

Comment:
The number is only important once merged, not knowing before hand what order they will be merged makes the job impossible